runtime/util_core/j9argscan.c

```diff
@@ -111,8 +115,29 @@ void scan_failed(J9PortLibrary * portLibrary, const char* module, const char *sc
 uintptr_t scan_double(char **scan_start, double *result)
 {
 	char *endPtr = NULL;
-
+#if defined(WINDOWS)
+	_locale_t clocale = _create_locale(LC_ALL, "C");
+	if (NULL != clocale) {
+		*result = _strtod_l(*scan_start, &endPtr, clocale);
+		_free_locale(clocale);
+	} else {
+		*result = strtod(*scan_start, &endPtr);
+	}
+#elif defined(LINUX) || defind(OSX) || defined(LC_ALL_MASK) /* defined(WINDOWS) */
+	/* AIX 7.1+ has support for newlocale() */
+	locale_t clocale = newlocale(LC_ALL_MASK, "C", NULL);
+	if (NULL != clocale) {
+		locale_t orglocale = uselocale(clocale);
+		*result = strtod(*scan_start, &endPtr);
+		uselocale(orglocale);
+		freelocale(clocale);
+	} else {
+		*result = strtod(*scan_start, &endPtr);
+	}
+#else /* defined(WINDOWS) */
+	/* No support on AIX 6.1 or z/OS so far, but testing shows strtod() isn't locale dependent */
```

Comment:
Realized I tested it incorrectly. z/OS at least is locale specific so I need another solution.